manifests/d/denglihong2007/CRSim/3.1.2.0/denglihong2007.CRSim.installer.yaml

```
@@ -0,0 +1,20 @@
+# Created with Sundry.
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.12.0.schema.json
+
+PackageIdentifier: denglihong2007.CRSim
+PackageVersion: 3.1.2.0
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+  - RelativeFilePath: CRSim.exe
+UpgradeBehavior: install
+Dependencies:
+  PackageDependencies:
+    - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.9
+ReleaseDate: 2025-11-15
+Installers:
+  - Architecture: neutral
```

```suggestion
  - Architecture: x64
```

The verification pipeline on the x86 architecture threw an executable error, but I don't have an x86 device to test it, so I'll switch it back to x64 for now.
https://dev.azure.com/shine-oss/winget-pkgs/_build/results?buildId=240464&view=artifacts&type=publishedArtifacts